streams-contrib/streams-provider-facebook/src/main/java/org/apache/streams/facebook/provider/FacebookUserstreamProvider.java

```diff
@@ -241,7 +240,7 @@ void shutdownAndAwaitTermination(ExecutorService pool) {
   @Override
   public void prepare(Object configurationObject) {
 
-    executor = MoreExecutors.listeningDecorator(newFixedThreadPoolWithQueueSize(5, 20));
+    executor = newFixedThreadPoolWithQueueSize(5, 20);
```

Comment:
I don't think we're going to be able to do this everywhere, but apparently it's ok in this provider if there's no compile error